scripts/start_ngas.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
I created this file only to help me with my development environment. Rather than have a discussion around wether or not we should be using NGAS with docker I would probably prefer to withdraw this script from the pull request.